app/views/users/registrations/new.html.erb

```diff
@@ -0,0 +1,43 @@
+<div>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "users/shared/error_messages", resource: resource %>
+
+  <div class="normal-inputs">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "name", placeholder: "User Name", class: 'input' %>
+  </div>
+
+  <div class="normal-inputs">
+    <%= f.label :bio %><br />
+    <%= f.text_field :bio, autofocus: true, autocomplete: "bio", placeholder: "User Biography", class: 'input' %>
+  </div>
+
+  <div class="normal-inputs">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", class: 'input' %>
+  </div>
+
+  <div class="normal-inputs">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password", class: 'input' %>
+  </div>
+
+  <div class="normal-inputs">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password", class: 'input' %>
+  </div>
```

Comment:
- [ ] At the moment, new user registration fails as a result of some required fields such as name, bio, and photo as you can see in the attached image below 👇. Please ensure that the registration works seamlessly 🙏 
### NB: The next reviewer will check the other project requirements after you fix the above issue ☝️ 
- [ ] Kindly increase the width of the confirm password input so that it matches the width of the other inputs on the user registration page 🙏 

![image](https://github.com/rodrigopenayo98/Blog-App-Rod/assets/46839548/f4c6fb9d-f72b-42e6-9946-56a3eef0605d)